src/code/z_collision_check.c

```diff
@@ -2,12 +2,65 @@
 #include "vt.h"
 
 // draw red poly
-void func_8005B280(GraphicsContext* gfx, Vec3f* vA, Vec3f* vB, Vec3f* vC) {
-    func_8005B2AC(gfx, vA, vB, vC, 255, 0, 0);
+void func_8005B280(GraphicsContext* gfxCtx, Vec3f* vA, Vec3f* vB, Vec3f* vC) {
```

Comment:
This might require an update in functions.h as well

Reply:
fixed prototypes.